.github/workflows/nbtest.yml

```diff
@@ -18,6 +18,7 @@ jobs:
             pip3 install nbmake
         - name: Install package
           run: |
+            git config --global url."https://${{ secrets.PAPYRUS_ACCESS }}@github".insteadOf https://github
```

Comment:
We can just make it public.

Reply:
Just made it public.

Reply:
Included in [a27cdac](https://github.com/zincware/ZnNL/pull/121/commits/a27cdace065c2a59e12574bf57fdc39370782e21)